mrp_packaging_default/models/mrp_bom_line.py

```diff
@@ -27,11 +27,6 @@ class MrpBomLine(models.Model):
     def _compute_product_packaging(self):
         """Set the appropriate packaging for the product qty."""
         for one in self:
-            one.product_packaging_id = (
-                one.product_id.packaging_ids._find_suitable_product_packaging(
-                    one.product_qty, one.product_uom_id
-                )
-            )
```

Comment:
Instead of moving this logic to product onchange, could you please test this?

```suggestion
            one.product_packaging_id = (
                one.product_packaging_id._find_suitable_product_packaging(
                    one.product_qty, one.product_uom_id
                ) or
                one.product_id.packaging_ids._find_suitable_product_packaging(
                    one.product_qty, one.product_uom_id
                )
            )
```

If this works, it's better to keep logic inside the compute method, because it works for multi-records and works for direct ORM operations.